spring-cloud-starter-task-composedtaskrunner/pom.xml

```diff
@@ -75,6 +76,9 @@
                             <autoConfigClass>org.springframework.cloud.task.app.composedtaskrunner.ComposedTaskRunnerConfiguration.class</autoConfigClass>
                         </composedtaskrunner-task>
                     </generatedApps>
+                    <additionalAppProperties>
+                        <additionalAppProperty>spring.cloud.task.closecontextEnabled=true</additionalAppProperty>
```

Comment:
Could you share the reason for doing this now explicitly? I don't seem to find a related discussion or an issue on this.

Reply:
Certainly!  In Task 1.x we had closecontextEnabled set to true by default.  As of 2.x the user must set this value if they want to close the context at the end of the task execution.   So when were in the 1.x line of task for CTR it would terminate at the end of the task run.   But when we converted to using Task 2.0 and the graph had a splitter the job would complete but the task application would not end.  This solution closeContextEnabled so that the CTR will have the same behavior as it did in the 1.3 line.